.github/workflows/build.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         shell: cmd
         run: ./src/build_official.cmd
         env:
-          RuntimeTestsEnabled: ${{ github.ref == 'refs/heads/master' }}
+          RuntimeTestsEnabled: true
```

Comment:
As far as I can tell, the Github Actions build has always been running the RuntimeTests. So I guess there's an additional issue here if you didn't do that on purpose.

Reply:
I checked and the `RuntimeTestsEnabled` was set to `false` correctly at this point so the issue would have been in the batch files, presumably.

Reply:
Ok. I wanted to make sure you are aware that changing this line doesn't affect anything today.

Reply:
Yeah, it's this code in `test.cmd`, if `RuntimeTestsEnabled` is set to anything tests run.

```
@if "%RuntimeTestsEnabled%"=="" echo Build integration tests %_C%
@if not "%RuntimeTestsEnabled%"=="" set _T=test&echo Run integration tests %_C%
```

I'll fix it as part of this.